docs/GIRDER.rst

```diff
@@ -0,0 +1,33 @@
+====================
+Background on Girder
+====================
+
+
+Navigating in Girder
+--------------------
+
+You will see several links on the left hand side of the screen that will allow you to navigate. These links are:
+
+* ``Collections`` - shows all of the top level collections. The only important collection for this project is ``WSI DeID``.
+* ``Groups`` - shows the user groups. You can ignore this for the purposes of this project.
+* ``Users`` - shows all of the users in the current system. As an admin user, you can reset a user's password by going to their user account and selecting Edit User from the Actions menu.
+* ``Admin console`` - will only be visible to admin users. You should leave this section alone unless you know what you are doing.
+* ``Next Item`` - will navigate any user to the next available image to be processed in the WSI DeID workflow.
+
+
+User Types and Permissions
+--------------------------
+
+**Admin User:** The first registered user of a WSI DeID system will be an ``admin`` user and will have super-user privileges, meaning that the user can take any actions on the system. All subsequently created users will be regular, non-super-users, but will have the ability to use the redaction workflows.
```

Comment:
Is it worth mentioning that the admin user can promote other users to be admins, too?